ci-operator/step-registry/ipi/OWNERS

```diff
@@ -1,10 +1,10 @@
 approvers:
-- smarterclayton
-- wking
-- vrutkovs
-- abhinavdahiya
-- deads2k
-- crawford
-- ewolinetz
-- csrwng
-- staebler
```

Comment:
I'm assuming this was a mistake.

Reply:
@jstuever Thanks for looking at this. I have cleaned these up. I believe a symlink was changed and the `make update` regenerates the files. I have cleaned these up.